pkgs/applications/misc/ulauncher/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, fetchFromGitHub, python27Packages, substituteAll,
+  gnome3, gobject-introspection, wrapGAppsHook, gtk3, webkitgtk,
+  libnotify, keybinder3, libappindicator, intltool, wmctrl }:
+
+python27Packages.buildPythonApplication rec  {
+  pname = "ulauncher";
+  version = "4.3.2.r8";
+
+  src = fetchFromGitHub {
+      owner = "Ulauncher";
+      repo = "Ulauncher";
+      rev = version;
+      sha256 = "19n5a5m9r84mczx2zkpa00k6n46ky2shnpfvxcxj04pcrl89bijq";
+	};
+
+  nativeBuildInputs = with python27Packages;  [
+    distutils_extra
+    intltool
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    keybinder3
+    libappindicator
+    libnotify
+    gnome3.adwaita-icon-theme
+    gobject-introspection
+    webkitgtk
+    wmctrl
+  ];
+
+  propagatedBuildInputs = with python27Packages; [
+    dbus-python
+    notify
+    pygobject3
+    pyinotify
+    pysqlite
+    python-Levenshtein
+    pyxdg
+    websocket_client
+  ];
+
+  doCheck = false;
```

Comment:
Why were these disabled? I think you were only missing `pytest` and possible there's an extension.